src/index.html

```diff
@@ -5,12 +5,80 @@
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      >
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+      >
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+      >
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="POST"
+      data-qa="big"
+      >
+      <div class="search-bar">
+        <div class="big-search-img">
+          <svg
+            width="19"
+            height="19"
+            viewBox="0 0 19 19"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            >
```

Comment:
```suggestion
          >
```
on the same level as the opening one